ynr/apps/people/tests/test_person_models.py

```diff
@@ -47,6 +47,42 @@ def test_get_display_image_url(self):
         person = Person.objects.get()
         self.assertEqual(person.get_display_image_url(), url)
 
+    def test_all_person_images_have_a_timestamp(self):
+        """ensure that all person images have a timestamp"""
+        person = PersonFactory(name=faker_factory.name())
+        pi = PersonImage.objects.create_from_file(
+            filename=EXAMPLE_IMAGE_FILENAME,
+            new_filename="images/jowell-pilot.jpg",
+            defaults={
+                "person": person,
+                "source": "Taken from Wikipedia",
+                "copyright": "example-license",
+                "user_notes": "A photo of Tessa Jowell",
+            },
+        )
+        self.assertIsNotNone(pi.created)
+        self.assertIsNotNone(pi.modified)
+
+    def test_person_image_modified_timestamp(self):
+        """visit the person-view.html template and check the response for the modified timestamp"""
+        person = PersonFactory(name=faker_factory.name())
+        pi = PersonImage.objects.create_from_file(
+            filename=EXAMPLE_IMAGE_FILENAME,
+            new_filename="images/jowell-pilot.jpg",
+            defaults={
+                "person": person,
+                "source": "Taken from Wikipedia",
+                "copyright": "example-license",
+                "user_notes": "A photo of Tessa Jowell",
+            },
+        )
+        modified = pi.modified
+        response = self.app.get(f"/person/{person.id}", user=self.user)
+        self.assertIn(
```

Comment:
BST problems? I tried this again and changed the migration update and created default to `datetime.now(timezone.utc).isoformat()` but got the same error.

Reply:
Is this working now? CI seems ok?

Reply:
No, I don't think so - it's a mismatch in the time.

Reply:
I've pushed a commit that fixes this, but I'm a little confused as to why we're not getting the right timezone back from `pi.modified`. In my understanding, the model should store the timezone and return local time, but maybe that's not how it works.